articles/quickstart/spa/angular-next/index.yml

```diff
@@ -0,0 +1,51 @@
+title: Angular
+image: /media/platforms/angular.png
+author:
+  name: Steve Hobbs 
+  email: steve.hobbs@auth0.com
+  community: false
+alias:
+  - angular
+language:
+  - Javascript
+framework:
+  - Angular
+topics:
+  - quickstart
+contentType: tutorial
+useCase: quickstart
+show_releases: true
+beta: true
+sdk:
+  name: auth0-angular
+  url: https://github.com/auth0/auth0-angular
+snippets:
+  dependencies: application-platforms/angular2/dependencies
+  setup: application-platforms/angular2/setup
+  use: application-platforms/angular2/use
+seo_alias: angular
+default_article: 01-login
+articles:
+  - 01-login
+show_steps: true
+github:
+  org: auth0-samples
+  repo: auth0-angular-samples
+  branch: next
```

Comment:
this branch doesn't exist. From the sample's PR: "merge 16 commits into master from update/new-sdk"

Reply:
Oops! Will change.